Sources/XcodeProj/Scheme/XCScheme+StoreKitConfigurationFileReference.swift

```diff
@@ -0,0 +1,43 @@
+//
```

Comment:
@jcolicchio would you mind removing the headers to be consistent with the rest of the codebase?

Reply:
Nevermind, I'll merge and fix those before releasing the next version.